src/cascadia/Remoting/WindowManager.cpp

```diff
@@ -759,11 +403,11 @@ namespace winrt::Microsoft::Terminal::Remoting::implementation
     // - <none>
     // Return Value:
     // - <none>
-    winrt::fire_and_forget WindowManager::RequestHideNotificationIcon()
+    winrt::fire_and_forget WindowManager::RequestHideNotificationIcon(Remoting::Peasant peasant)
     {
         auto strongThis{ get_strong() };

```

Comment:
Now that this doesn't rely on `this` anymore, you can remove the `auto strongThis{ get_strong() };`. Same below.